### PR TITLE
fix opensip3.1 docker image

### PR DESCRIPTION
**Pull request by daxiondi**
Because of version 3.1, the listen is modified to socket
so the following changes are made.
looking forward to your review

### CODE REVIEW

File: Dockerfile-3.1 [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    apt-get -y install opensips-dialplan-module \
+    ;fi
+
+RUN rm -rf /var/lib/apt/lists/*
+
+EXPOSE 5060/udp
+
+COPY run-3.1.sh /run-3.1.sh
+
+ENTRYPOINT ["/run-3.1.sh"]
```
Comment (razvancrainea) [inline Dockerfile-3.1:53]: I'm not sure adding a new `Dockerfile` is the right approach, I'd rather create a different branch (3.0), and make the changes on master. I believe this is easier for people to work this out.

File: Makefile [hunk 2]
```diff
@@ -24,4 +37,4 @@ build:
 		.
 
 start:
-	docker run -d --name $(NAME) opensips/opensips:$(OPENSIPS_DOCKER_TAG)
+	docker run -p 5060:5060/udp -d --name $(NAME) opensips/opensips:$(OPENSIPS_DOCKER_TAG)
```
Comment (razvancrainea) [inline Makefile:40]: let's not hardcode this, instead make it configurable as extra docker params

File: Dockerfile-3.1 (hunk 1 above)
Comment (daxiondi) [reply Dockerfile-3.1:53]: Dockerfile is completely correct. I have tested it. Creating a 3.0 branch is optional, but not required

**Comment by daxiondi** [comment]
I have fixed the problem you said. If there are any problems, please continue to review. I would like to make open source contribution to opensips. Thank you for your review

**Comment by razvancrainea** [comment]
Thanks for you work here, but since we're no longer supporting OpenSIPS 3.0, this patch basically becomes obsolete.
I have pushed the master to use 3.1, but one can still use it to generate 3.0 or 2.4 using the `OPENSIPS_VERSION` variable.

**Pull request closed without merging**